lua/neotest-jest/jest-util.lua

```diff
@@ -2,8 +2,8 @@ local M = {}
 
 local lib = require("neotest.lib")
 local util = require("neotest-jest.util")
-
-local rootPackageJsonPath = vim.fn.getcwd() .. "/package.json"
+local uv = vim.loop
+local rootPackageJsonPath = uv.cwd() .. "/package.json"
```

Comment:
Please use the `compat` module for this as you otherwise break support for older versions.